readme/api/references/plugin_manifest.md

```diff
@@ -15,6 +15,7 @@ Name | Type | Required? | Description
 `repository_url` | string | No | Repository URL where the plugin source code is hosted.
 `categories` | string[] | No | [Categories](#categories) that describes the functionality of the plugin. 
 `screenshots` | Screenshot[] | No  | [Screenshots](#Screenshot) are used for listing on Joplin Plugin website.
+`icons` | Icons | No | Icons are mainly used for be displaying on the desktop app plugins page and on Joplin Plugin Website.
```

Comment:
Add a link to the documentation, like the other properties

Reply:
"Icons are used for listing on the desktop app plugins page and on the Joplin Plugin website."